packages/flame/lib/src/events/messages/event.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flame/src/components/component.dart';
+import 'package:meta/meta.dart';
+
+/// Base class for a variety of input events, such as tap events, drag events,
+/// keyboard events, etc.
+///
+/// This base class offers only simple common functionality; please see the
+/// concrete [Event] subclasses for the information about each individual event
+/// and the circumstances when they occur.
+abstract class Event {
+  /// Flag that can be used to indicate that the event was handled by one of the
+  /// components.
+  ///
+  /// This flag is neither set nor read by Flame. Instead, it can be set by the
+  /// user in a component that handles an event, and then read by the user in a
+  /// different component, or at the root Game level.
+  bool handled = false;
+
+  /// If this flag is false (default), the event will be delivered to the first
+  /// component that can handle it. If that component sets this flag to true,
+  /// the event will propagate further down the component tree to other eligible
+  /// components.
+  bool continuePropagation = false;
```

Comment:
Currently this defaults to true, but maybe it's better to default to false.

Reply:
I don't understand, the default is false here, and it says so in the doc-comment too.

Reply:
Yes, but currently for the `Tappable` it is true.

Reply:
Ok, so no change's needed for this PR, right?

Reply:
No, I think this is good, because I think this is how it usually is in these kind of event systems, that it by default stops after the first "thing" that it hits.